code/modules/hydroponics/biogenerator.dm

```diff
@@ -1,337 +1,354 @@
+/// The base amount of plants that can be stored before taking our matter bin into account.
+#define BASE_MAX_STORABLE_PLANTS 40
+
 /obj/machinery/biogenerator
-	name = "Biogenerator"
+	name = "biogenerator"
 	desc = "Converts plants into biomass, which can be used to construct useful items."
 	icon = 'icons/obj/biogenerator.dmi'
 	icon_state = "biogen-empty"
-	density = 1
-	anchored = 1
+	density = TRUE
+	anchored = TRUE
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 40
-	var/processing = 0
-	var/obj/item/reagent_containers/glass/beaker = null
-	var/points = 0
-	var/menustat = "menu"
+	/// Is the biogenerator curretly grinding up plants?
+	var/processing = FALSE
+	/// The container that is used to store reagents from certain products.
+	var/obj/item/reagent_containers/glass/container
+	/// The amount of biomass stored in the machine.
+	var/biomass = NONE
+	/// Used to modify the cost of producing items. A higher number means cheaper costs.
 	var/efficiency = 0
+	/// Used to modify how much biomass is produced by grinding plants. A higher number means more biomass.
 	var/productivity = 0
-	var/max_items = 40
+	/// A list of plants currently stored plants in the biogenerator.
+	var/list/stored_plants = list()
```

Comment:
It would make sense, however it's not necessary. The reason I didn't is simply because I didn't like how 
`var/list/obj/item/reagent_containers/food/snacks/grown/stored_plants = list()` looks.